packages/ui/src/components/select/NetworkSelection.tsx

```diff
@@ -14,31 +21,179 @@ const NetworkSelection = () => {
+      onChange={(event) => handleNetworkSelection(event as SelectChangeEvent<string>)}
+      MenuProps={MenuPropsStyles}
+    >
+      <ListSubheader>Polkadot & Parachains</ListSubheader>
+      {renderNetworks(polkadotNetworksAndParachains)}
+      <ListSubheader>Kusama & Parachains</ListSubheader>
+      {renderNetworks(kusamaNetworksAndParachains)}
+      <ListSubheader>Testnets</ListSubheader>
+      {renderNetworks(testChains)}
+    </SelectStyled>
   )
 }
 
+const MenuPropsStyles = {
+  sx: {
+    marginTop: '.75rem',
+
+    '.MuiPaper-root': {
+      marginTop: '2px',
+      boxShadow: 'none',
+      border: 'none',
+      outline: `1px solid ${theme.custom.text.borderColor}`,
+
+      ul: {
+        border: 'none'
+      }
+    },
+
+    '.MuiMenuItem-root': {
+      maxWidth: '100%',
+      padding: '0.75rem'
+    },
+
+    '.MuiList-root': {
+      columnFill: 'balance',
+      padding: 0,
+      border: `1px solid ${theme.custom.text.borderColor}`,
+      borderRadius: '0.5rem'
+    },
+
+    '.MuiListSubheader-root': {
+      columnSpan: 'all'
+    },
+
+    [theme.breakpoints.up('xs')]: {
+      '.MuiList-root': {
+        columns: '160px 2'
+      }
+    },
+
+    [theme.breakpoints.up('sm')]: {
+      '.MuiList-root': {
+        columns: '180px 2'
+      }
+    },
+
+    [theme.breakpoints.up('md')]: {
+      '.MuiList-root': {
+        columns: '200px 2'
+      }
+    }
+  }
+}
+
+const SelectStyled = styled(SelectMui)`
+  display: inline-flex;
+  width: 100%;
+  background: ${({ theme }) => theme.palette.primary.white};
+  outline: 1.5px solid ${({ theme }) => theme.custom.text.borderColor};
+  text-transform: capitalize;
+  border-radius: ${({ theme }) => theme.custom.borderRadius};
+  cursor: pointer;
+
+  @media (min-width: ${({ theme }) => theme.breakpoints.values.md}px) {
+    margin-left: 1rem;
+    max-width: fit-content;
+  }
+
+  .MuiSelect-select {
+    display: flex;
+    padding-right: 2.5rem !important;
+
+    @media (min-width: ${({ theme }) => theme.breakpoints.values.md}px) {
+      padding-right: 2rem !important;
+    }
+
+    div:last-child {
+      @media (min-width: ${({ theme }) => theme.breakpoints.values.md}px) {
+        display: none;
+      }
+    }
+  }
+
+  .MuiList-root {
+    columns: 100vw 3;
+    column-fill: balance;
+
+    @media (min-width: ${({ theme }) => theme.breakpoints.values.md}px) {
+      columns: 100px 2;
+      column-fill: auto;
+    }
+  }
+
+  svg {
+    color: ${({ theme }) => theme.custom.text.black};
+    width: 1.25rem;
+    height: 1.25rem;
+    top: calc(50% - 0.65em);
+  }
+
+  .MuiSelect-icon {
+    right: 9px;
+  }
+
+  .MuiOutlinedInput-notchedOutline {
+    border: none;
+  }
+`
+
+const MenuItemStyled = styled(MenuItem)`
+  text-transform: capitalize;
+  padding: 0.75rem;
+  max-width: 9.1875rem;
+  box-sizing: content-box;
+`
+
+const ImgStyled = styled('img')`
+  width: 1.25rem;
+  height: 1.25rem;
+  margin-right: 0.5rem;
+  border-radius: 50%;
+  max-height: 2rem;
```

Comment:
Should we have a much bigger logo then? I've had nice results from a `2rem` and some padding changes on the `.MuiSelect-select` see next comment.
```suggestion
  width: 2rem;
  height: 2rem;
  margin-right: 0.5rem;
  border-radius: 50%;
```

Reply:
thank you for the suggestion! 2 rem is probably too big, 1.75 looks better for me